### ISSUE TITLE
eks-rolling-update is going into crashloopfailure

### ISSUE THREAD

**Issue by spacetimesv** [hellofresh/eks-rolling-update#126]
eks-rolling-update 1.20.2 is going into crashloopfailure and is not stable and also shuts down cluster-autoscaler. This is on EKS 1.20 version cluster.

Logs below
2022-03-06 07:48:03,281 INFO     Exiting since ASG healthcheck failed
2022-03-06 07:48:03,282 ERROR    ASG healthcheck failed
2022-03-06 07:48:03,282 ERROR    *** Rolling update of ASG has failed. Exiting ***
2022-03-06 07:48:03,282 ERROR    AWS Auto Scaling Group processes will need resuming manually
2022-03-06 07:48:03,282 ERROR    Kubernetes Cluster Autoscaler will need resuming manually

Can you please suggest what is the supported version for EKS 1.20.2 and kindly provide the full deployment yaml. thanks

**Comment by dgem** [comment]
I don't think it is intended to deploy this into a cluster, after all your eks_rolling_update container could be on a node that is then scheduled to be terminated... or perhaps I'm misinterpreting what you mean by saying "eks-rolling-update 1.20.2 is going into crashloopfailure"
